src/features/DeFiZap/components/DashboardZapCTA.tsx

```diff
@@ -28,11 +40,31 @@ const DashboardZapCTA = ({ className }: any) => {
       headingRight={<DeFiZapLogo />}
       className={className}
     >
-      <CTAContent>
-        {Object.values(zapConfig).map((zap) => (
-          <ZapCard config={zap} key={`key-${zap.key}`} />
-        ))}
-      </CTAContent>
+      <>
+        <SubHeaderText>
+          <Trans
+            id="ZAP_DASHBOARD_PANEL_SUB_HEADER"
+            variables={{
+              $readMore: () => (
+                <LinkOut
+                  showIcon={false}
+                  inline={true}
+                  fontSize={FONT_SIZE.BASE}
+                  fontColor={COLORS.BLUE_BRIGHT}
+                  link={getKBHelpArticle(KB_HELP_ARTICLE.HOW_TO_USE_ZAPPER)}
+                  text={translateRaw('READ_MORE')}
+                />
+              )
+            }}
+          />
+        </SubHeaderText>
```

Comment:
You can use Markdown in translation file to define the link

Reply:
I think this is a better way, hardcoding links in translations file is not the best way, and this way the link could also be customized more easily. For example, changing color, adding icon, and so on.

Reply:
Hardcoding this in translation files, it would require changing all languages links, this way only one line needs to change.

Reply:
Yes it adds more flexibility but won’t follow global style, we're trying to uniform the style to have less style rules declarations within small components

It don't even needs to be hardcoded, you can keep the variables but define the link as `[$text]($link)` in translation file so no need to change all languages links :)